src/styles/components/_card.scss

```diff
@@ -101,12 +121,47 @@
   }
 }
 
+@mixin bsds-modifier-error-text {
```

Comment:
Drop opening brace

We may want to pull this up to _vars-utility since we could theoretically use this in other components when there are mismatched modifiers